games/gopsHelpers.js

```diff
@@ -72,17 +78,78 @@ function roundWinner(instance) {
   }
 };
 
+//Awards points to the winner of the round. Takes roundWinner as an argument.
 function awardPoints(instance, id) {
 
   if (id) {
     instance[id].points += instance.prizePool[0];
     instance.prizePool.splice(0, 1);
+    reset(instance);
   } else {
     let message = "Round nulled, two or more players played same card."
     instance.prizePool.splice(0, 1);
+    reset(instance);
+    return message;
   }
 }
 
+//Once winner has been decided, resets hand of each player to zero.
+function reset(instance) {
+
+  for (let player in instance) {
+    if(instance[player].readyCard !== undefined){
+      instance[player].readyCard = 0
+    }
+  }
+};
+
+//At the begginning of every round, will check to see if every player has played. Status conditional -- needs to run
+//before checking a winner.
+function readyCheck(instance) {
+
+  let ready = true;
+  for (let player in instance) {
+    if(instance[player].readyCard === 0) {
+      ready = false;
+    }
+  }
+  return ready;
+}
+
+//Cheecks to see if the game is still playable based on the number of cards in the prize pool. At zero, returns false.
+function gameCheck(instance){
+
+  let status = true;
+  if (instance.prizePool.length === 0) {
+    status = false
+  }
+  return status;
+}
+
+//Will run after having checked if game is done. If it is, will dfind the player with the highest points and return that.
+function endGame (instance) {
+
+  let gameWinner = 0;
+  let totalPoints = 0;
+
+  for (let player in instance) {
+    if ((instance[player].points) > totalPoints){
+      totalPoints = instance[player].points;
+      gameWinner = player;
+    }
+  }
+  return gameWinner;
+}
+
+// Will run after a player has joined the round. Returns true if the maximum number of players ahave been reached.
+function startCheck(instance) {
+  status = false;
+  if (instance.playerCount === instance.maxPlayers){
```

Comment:
It's not accessing the DB to get that number, for the time being it's being hardcoded in, though I'll rename it autostart players for clarity! :)